atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -48,6 +48,13 @@
 
     String TYPE = "cassandra";
 
+    /**
+     * These are only the initial 'contact points' that will be used in connecting with the cluster. AtlasDB will
+     * subsequently discover additional hosts in the cluster. (This is true for both Thrift and CQL endpoints.)
+     *
+     * This value, or values derived from it (e.g. the number of Thrift hosts) MUST NOT be used to ascertain the
```

Comment:
maybe change the wording to "must only be used on KVS initialization to generate the initial connection(s) to the cluster"?

Reply:
Tf we haven't already I think we should audit use of this method across the codebase and ensure it aligns with the expectations of this javadoc. The method below we've altered in this PR, even after these changes actually calls this method when it should not (even if it is only for logging purposes, it is still incorrect).

Reply:
Yep, changed. I did do this audit actually, though agree that the log message makes more sense when it actually uses the servers discovered in the cluster.